__PySiddhi4Proxy/threadfix_c_code/makefile

```diff
@@ -1,18 +1,41 @@
+# ---------------------------------------------------------------------------
+#  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please fix the License header year in all the places

Reply:
This file was added in 2017. That's why I added 2017 as the license year